stdlib/3/urllib/request.pyi

```diff
@@ -34,6 +34,7 @@ def pathname2url(path: str) -> str: ...
 def getproxies() -> Dict[str, str]: ...
 def parse_http_list(s: str) -> List[str]: ...
 def parse_keqv_list(l: List[str]) -> Dict[str, str]: ...
+def proxy_bypass(host: str) -> Any: ...  # Undocumented
```

Comment:
Returns `int` as far as I can see (probably in place of a `bool`).

Reply:
Yes it returns `int` (`0` or `1`).

However docstring specifies: `Return True, if host should be bypassed`.

The actual usage of the function in the module shows the typical usage like `if proxy_bypass(...):`.

This lead be to conclusion that `Any` is the proper return type, as it allows bool-like checks and doesn't specify the return type precisely that could produce warnings in runtime type checkers like `typeguard`.

Reply:
Sounds reasonable.